src/components/countyDashboard.tsx

```diff
@@ -41,12 +46,12 @@ const CountyDashboard: React.FC = () => {
     setCurrCounty(value);
   };
 
-  const stateCounties = countyData[currState as CountyName];
+  const handleDataTypeChange = (value: DataType) => {
+    setCurrData(value);
+    setCurrLink(dataLinks[value]);
+  };
 
-  // should change when currState is updated. Shows map of counties in a state
-  const stateCountiesDashboardURL = 'https://datawrapper.dwcdn.net/q9LZ6/3/';
-  // should change when currCounty is updated. Shows map of a county
-  const countiesDashboardURL = 'https://datawrapper.dwcdn.net/EviBb/2/';
+  const stateCounties = countyData[currState as CountyName];
```

Comment:
not advisable to use `as`
```suggestion
  const stateCounties = countyData[currState];
```